test/e2e/libpod_suite_test.go

```diff
@@ -141,6 +143,10 @@ func PodmanCreate(tempDir string) PodmanTest {
 	if os.Getenv("STORAGE_OPTIONS") != "" {
 		storageOptions = os.Getenv("STORAGE_OPTIONS")
 	}
+	cgroupManager := CGROUP_MANAGER
+	if os.Getenv("CGROUP_MANAGER") != "" {
+		storageOptions = os.Getenv("CGROUP_MANAGER")
```

Comment:
should this be cgroupManager and not storageOptions?

Reply:
Good catch, but that doesn't appear to be the root of our issues